MediaBrowser.WebDashboard/dashboard-ui/css/librarybrowser.css

```diff
@@ -216,6 +216,7 @@
 
 .itemOverview {
     white-space: pre-wrap;
+    font-size: 20px;
```

Comment:
We generally try to avoid fixed font sizes, and instead just let the sizes flow from the top of the document down.